### PR TITLE
Gibs now rot and generate miasma

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Gibs will now rot if not cleaned, and produce small amounts of miasma, approximately
equal to a quarter of a corpse.
/:cl:

Miasma is fun. Gives the janitor more of a reason to do work. It will only become
dangerous if you have a lot of gibs in a small space. So better keep those Hopline shutters
open.

### CODE REVIEW

**Comment by Steelpoint** [comment]
This is a nice change, although I do think miasma needs to be more prominent since I've never seen miasma become a issue at any point.

**Comment by MACIEKBAKI** [comment]
Well you know miasma factories.. you could always set up a miasma factory and put it in the distro loop... crazy antag idea right there!!

**Comment by Potato-Masher** [comment]
Literally, the only time I have ever seen Miasma come into play was yesterday when someone made a gas chamber with Miasma that would keep you throwing up for a solid 10 minutes after you got out.

**Comment by Arianya** [comment]
We Dwarf Fortress Now

File: code/game/objects/effects/decals/cleanable/humans.dm [hunk 1]
```diff
@@ -54,9 +54,35 @@
...
+	name = "rotting [initial(name)]"
+	desc += " It smells terrible."
+	START_PROCESSING(SSobj, src)
+
+/obj/effect/decal/cleanable/blood/gibs/process()
+	var/turf/open/T = get_turf(src)
+	if(!istype(T))
+		return
+
+	var/list/cached_gases = T.air.gases
```
Comment (Cyberboss) [inline code/game/objects/effects/decals/cleanable/humans.dm]: Use T.return_air() instead of T.air

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/decals/cleanable/humans.dm [hunk 2]
```diff
@@ -54,9 +54,24 @@
...
 /obj/effect/decal/cleanable/blood/gibs/Initialize(mapload, list/datum/disease/diseases)
 	. = ..()
 	reagents.add_reagent("liquidgibs", 5)
+	if(already_rotting)
+		start_rotting(rename=FALSE)
+	else
+		addtimer(CALLBACK(src, .proc/start_rotting), 2 MINUTES)
+
+/obj/effect/decal/cleanable/blood/gibs/Destroy()
+	STOP_PROCESSING(SSobj, src)
```
Comment (Cyberboss) [inline code/game/objects/effects/decals/cleanable/humans.dm]: Unnecessary?

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/decals/cleanable/humans.dm (hunk 2 above)
Comment (coiax) [reply code/game/objects/effects/decals/cleanable/humans.dm]: Yep, nice catch.

**Comment by Jordie0608** [comment]
You are the weakest link coiax.

**Pull request merged**